build/tools/CFamilyJarPreProcessor/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Not sure if it's worth committing it. It might be confusing to other devs if they run it and not realize where this is coming from.